packages/dx-react-scheduler/src/plugins/all-day-panel.tsx

```diff
@@ -59,6 +59,9 @@ class AllDayPanelBase extends React.PureComponent<AllDayPanelProps, AllDayPanelS
     this.setState({ rects, elementsMeta: cellElementsMeta });
   });
 
+  getMemoMessageFormatter = memoize(messages =>
+    getMessagesFormatter({ ...defaultMessages, ...messages }));
+
```

Comment:
Can we extract this function to `core` package? I think we can use it in all of our packages.

Reply:
Yes, we can